### PR TITLE
Added deployment settings for multiple component record

### PR DESCRIPTION
**Pull request by anurag-ks**

### CODE REVIEW

File: modules/s3cfg.py [hunk 1]
```diff
@@ -2025,6 +2025,12 @@ def get_cms_show_titles(self):
         """
         return self.cms.get("show_titles", False)
 
+    def get_cms_multiple(self):
+        """
+            Whether there should exist one component record per master record
```
Comment (hitesh96db) [inline modules/s3cfg.py]: This is what "multiple" generally means, but the description should tell something more specific about the CMS module(where is it possibly being used ?).

File: modules/s3db/cms.py [hunk 2]
```diff
@@ -358,13 +358,11 @@ def model(self):
...
                        cms_post_organisation = {"joinby": "post_id",
-                                                # @ToDo: deployment_setting
-                                                "multiple": False,
+                                                "multiple": settings.get_cms_multiple(),
                                                 },
 
                        cms_post_organisation_group = {"joinby": "post_id",
-                                                      # @ToDo: deployment_setting
-                                                      "multiple": False,
+                                                      "multiple": settings.get_cms_multiple(),
```
Comment (hitesh96db) [inline modules/s3db/cms.py]: I don't think these settings should be common.
@somayjain  @arnavsharma93 ?

File: private/templates/default/config.py [hunk 3]
```diff
@@ -435,6 +435,8 @@
 #settings.cms.organisation_group = "post_organisation_group.group_id"
 # Uncomment to use person_id instead of created_by in Newsfeed
 #settings.cms.person = "person_id"
+# Uncomment to use multiple component record per master record
```
Comment (hitesh96db) [inline private/templates/default/config.py]: Description should be more specific to the module.

**Comment by hitesh96db** [comment]
That was quick ! ;)

**Comment by anurag-ks** [comment]
Hey i have update the code
please review it.

File: private/templates/default/config.py [hunk 4]
```diff
@@ -435,6 +435,10 @@
 #settings.cms.organisation_group = "post_organisation_group.group_id"
 # Uncomment to use person_id instead of created_by in Newsfeed
 #settings.cms.person = "person_id"
+# Uncomment to enable multiple Organisations per Posts
+#settings.cms.mulitple_organisations = True
```
Comment (hitesh96db) [inline private/templates/default/config.py]: Typo(multiple)

File: modules/s3cfg.py [hunk 5]
```diff
@@ -2025,6 +2025,18 @@ def get_cms_show_titles(self):
         """
         return self.cms.get("show_titles", False)
 
+    def get_cms_multiple_organisation(self):
+        """
+            Whether there should exist mulitple Org components per Posts
+        """
+        return self.cms.get("multiple_organisations", False)
```
Comment (hitesh96db) [inline modules/s3cfg.py:2032]: If there's a setting like 

<pre> settings.cms.multiple_organisations = True </pre>

then the function definition should be like this

<pre> def get_cms_multiple_organisations(self): </pre>

and not

<pre> def get_cms_multiple_organisation(self): </pre>

Notice the 's' after organisation( 's' is there in the variable name so it's included in the function definition as well ).
I am pretty sure this is a convention being followed ;)

**Comment by hitesh96db** [comment]
Some minor changes, otherwise I think it's good :+1:

**Comment by anurag-ks** [comment]
Done the changes.

**Comment by somayjain** [comment]
The new deployment settings should be added to the default config file too as comments for reference. Like it's been done here - https://github.com/edengci/eden/blob/master/private/templates/default/config.py#L415

Looks good otherwise

**Comment by anurag-ks** [comment]
I have added it to the config file already
https://github.com/anurag-ks/eden/blob/master/private/templates/default/config.py#L439 and the line after that. @somayjain . I hope it's fine.

**Comment by somayjain** [comment]
Looks fine to me.

**Comment by anurag-ks** [comment]
@somayjain could you please review this task then - https://www.google-melange.com/gci/task/view/google/gci2014/5847257765117952

**Comment by anurag-ks** [comment]
Please review the task.

**Comment by anurag-ks** [comment]
@hitesh96db are done making the branch, or could you please mark the task as complete I want to move on to the next task. I'll send the PR as soons as you make the new branch

**Comment by hitesh96db** [comment]
Yeah, I'll mark it as done.

**Pull request merged**